.gitignore

```diff
@@ -6,4 +6,4 @@ jwtRS256.key
 jwtRS256.key.pub
 keygen.sh
 database.json
-.env*
+.env*
```

Comment:
Haha, interesting, what is that? a \n ? it would lead to problem?

Reply:
No, sorry I just removed it by accident in a commit